### PR TITLE
Add several new Kujira tokens

### PR DESCRIPTION
**Pull request by Rorcual-Nodes**
Added tokens to assetlist.json :

- xUSK
- xMNTA
- stATOM
- stOSMO
- wBTC.axl
- Several LP Tokens

Images for xTOKENS and LPs

### CODE REVIEW

File: kujira/assetlist.json [hunk 1]
```diff
@@ -1155,6 +1155,1134 @@
...
+                "denom": "xusk",
+                "exponent": 6
+            }
+        ],
+        "base": "factory/kujira1w4yaama77v53fp0f9343t9w2f932z526vj970n2jv5055a7gt92sxgwypf/urcpt",
+        "name": "Ghost Vault USK",
+        "display": "xusk",
+        "symbol": "xUSK",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/ghost/xusk.png"
```
Comment (JeremyParish69) [inline kujira/assetlist.json:1176]: Please don't create a /ghost/ sub-directory within /images/

File: kujira/assetlist.json [hunk 2]
```diff
@@ -1155,6 +1155,1134 @@
...
+            },
+            {
+                "denom": "statom",
+                "exponent": 6
+            }
+        ],
+        "base": "ibc/0306D6B66EAA2EDBB7EAD23C0EC9DDFC69BB43E80B398035E90FBCFEF3FD1A87",
+        "name": "stATOM",
+        "display": "statom",
+        "symbol": "stATOM",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:1224]: Please define `traces` for all ibc-transferred assets

File: kujira/assetlist.json [hunk 3]
```diff
@@ -1155,6 +1155,1134 @@
...
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.svg"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/statom.png",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:1231]: Please include image_sync for all images referencing images stored in another chain's /images/ directory.

File: kujira/assetlist.json [hunk 4]
```diff
@@ -1155,6 +1155,1134 @@
...
+        "name": "LP Bow: CHEQ-MNTA",
+        "display": "lp cheq-mnta",
+        "symbol": "LP CHEQ-MNTA",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:1418]: Due to the different backgrounds (navy vs transparent), bow.svg and bow.png are not visually identical and must be registered under separate image objects. So only one of these can be primary and may go into logo_URIs

File: kujira/assetlist.json [hunk 5]
```diff
@@ -1155,6 +1155,1134 @@
...
+    },
+    {
+        "description": "LP Bow: CHEQ-MNTA",
+        "denom_units": [
+            {
+                "denom": "factory/kujira15w6jhxsv924mn9cfr0qg9z36hltmw4rzd68x40d9re6hzyhslumqjw9zqd/ulp",
+                "exponent": 0
+            },
+            {
+                "denom": "lp cheq-mnta",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:1404]: Please do not include spaces in the denom

**Comment by Rorcual-Nodes** [comment]
@JeremyParish69  thank you so much for your review comments. First time submitting assets to the registry, Proceeded to rectify:

- Added `traces` for IBC denoms.
- Removed images subdirectory.
- Removed conflict on BOW images due to different background.
- Removed spaces in denoms.
- Added ìmage_sync` for third party chains.

File: kujira/assetlist.json [hunk 6]
```diff
@@ -1,1160 +1,2264 @@
...
+            "base": "factory/kujira12tpc335t72qtrjh64ccdydzsr9qtswkqn72cyvfj4d5hmtgynphsv4xjtu/ulp",
+            "name": "LP Bow: SHD-MNTA",
+            "display": "lp-shd-mnta",
+            "symbol": "LP SHD-MNTA",
+            "logo_URIs": {
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+            },
+            "images": [
+                {
+                    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
```
Comment (JeremyParish69) [inline kujira/assetlist.json:1384]: For cases where you use bow.svg, can we also define bow.png as a new image object? i.e., 
```
"images": [
  {
    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
  },
  {
    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png"
  }
]
```

Comment (JeremyParish69) [reply kujira/assetlist.json:1384]: Or else remove bow.png because it is unused

**Comment by JeremyParish69** [comment]
Can you please correct the formatting back to 2 spaces? It is very difficult to compare the differences when the formatting is changed

File: kujira/assetlist.json (hunk 6 above)
Comment (Rorcual-Nodes) [reply kujira/assetlist.json:1384]: Yes, bow.png was removed from all items on the last commit.

**Comment by Rorcual-Nodes** [comment]
Checked bow.png was removed and indentation back to 2.

File: kujira/assetlist.json (hunk 6 above)
Comment (JeremyParish69) [reply kujira/assetlist.json]: It was removed from the code but the file is still there under images. I'm saying we should make use of it (by specifying it as an alternative, or else remove the file from /images/)

Comment (Rorcual-Nodes) [reply kujira/assetlist.json]: Ah yeah! My bad. Removed the file. Cheers

**Pull request merged**